### DISCUSSION TITLE
Automatic vertical scrolling bar on every page in Serenity 6.9

### DISCUSSION THREAD

**Discussion by flashyjunior** [serenity-is/Serenity#7122] [Help]
Please I need help to take away the automatic vertical scrolling bar on every page, this is not like that in another project i have using version Serenity 5.0, please help

**Version 6.9 with vertical scrollbar, you have to scroll down before seeing the grid footer**

![image](https://github.com/serenity-is/Serenity/assets/7760804/17d458bf-82f3-428b-a960-c8b5906dd575)

**Version 5.0 with No vertical scroll bar**

![image](https://github.com/serenity-is/Serenity/assets/7760804/d64f907a-cb30-4b43-ad0c-44c25bfcb26a)

**Comment by LSNicholls** [comment]
Hi,

Check this out https://github.com/orgs/serenity-is/discussions/7102
